### ISSUE TITLE
Request - Educational Collection Attribute Marker

### ISSUE THREAD

**Issue by Kmullineaux** [ArctosDB/arctos#8347]
- [ ] new feature or otherwise a good idea to make Arctos even better

**Describe what you're trying to do** 

The NHSM has a permanent collection, but we also have educational collection objects that lack data but are cataloged in Arctos for tracking and management purposes. I would like to create an attribute that can be used to mark educational collection items and be used to filter them out when querying the collection. Also, if it's possible, I'd like to make them private from public searching. 

Thank you!

**Comment by mkoo** [comment]
I am puzzling over your request and think I need more information and examples (use cases). If you prefer please contact me for a zoom meeting or let me know if you can attend the next Arctos office hour and I will try to attend too!

**Comment by campmlc** [comment]
This sounds like a good use case for an encumbrance to mask record from public view which can be named educational collection. Not sure about filtering - if you mean from.public searches, the encumbrance would work. For curatorial searches, you could query from a public login.

**Comment by mkoo** [comment]
One could treat as encumbrance but it has some "costs" (curatorial management ones are the first that come to mind), which may not be a sustainable in the long run. Do they have to be in the same collection as specimens at all?

**Comment by Kmullineaux** [comment]
@mkoo and @campmlc Thank you for your replies! The educational items are not accessioned, but for management purposes, I've cataloged them in Arctos and attached them to placeholder accession records that provide information on their acquisition. For example, we may have a taxidermy duck that we use for education and display without data. In this case, we'd want to be able to track it internally, but the public/researchers browsing our records online would have no use for it. Therefore, we want to make it unavailable or at least distinguishable. I can see the benefits of an encumbrance for this.  

Regarding whether they have to be in the same collection, I've puzzled over this as well. I don't see why they need to be. However, some are already cataloged under their related collections. Would that cause an issue if we moved them to a new educational collection?

**Comment by Kmullineaux** [comment]
@dustymc Yes, good point. The duck is not a phantom without any information! I will continue puzzling on this with my team members to see what works best for us. If we decide on a teaching collection in the future, is it possible to change the associated collection of an already-cataloged object, or would we need to catalog it again and delete the old record?

**Comment by dustymc** [comment]
>  change the associated collection

That's not user-accessible but we can help,  sooner (before the data finds ways to get complex) will be smoother.

**Comment by mkoo** [comment]
I like the preference of using a dedicated accession or data loan (basically a transaction) to track usage and groupings of records in your use case. Also for filtering out these records, are you using record_type consistently? 

For example: https://arctos.database.museum/info/ctDocumentation.cfm?table=ctcataloged_item_type the educational material could be ```MaterialEntity``` vs ```PreservedSpecimen ``` 

If you'd like to continue discussion, I can transfer this to a dedicated discussion thread; otherwise I will close

**Comment by Kmullineaux** [comment]
@mkoo my teammates decided to not put educational materials for the other collections into Arctos to avoid the confusion over accessioned v. non-accessioned materials. 

I believe they want to store those materials and their data elsewhere and delete the educational records in Arctos that apply for the NHSM:Bird collection. I'm still waiting on them to make final choices so I haven't removed any records at this time. If there were a way to put them under a different transaction type, I could present that. I have been using record type consistently, but maybe not in the most perfect way.

**Comment by mkoo** [comment]
Thanks for the update. Another thing to consider depending if you have a large number of items to catalog or manage, is to have a separate collection for Teaching materials (eg NHSM:Teach) as a way to separate out nonaccessioned, non-loaning material. There is still value it seems to find that one species of duck that's mounted for someone.

Please file a new issue with new ideas or updates.
